kategory/src/main/kotlin/kategory/instances/NumberInstances.kt

```diff
@@ -39,4 +39,4 @@ object FloatMonoid : Monoid<Float>, Semigroup<Float> by SGFloat, GlobalInstance<
     override fun empty(): Float = .0f
 }
 
-private val SGFloat: Semigroup<Float> = NumberSemigroup(Float::plus)
\ No newline at end of file
+private val SGFloat: Semigroup<Float> = NumberSemigroup(Float::plus)
```

Comment:
yeah, I just moved them to another file but my point as mentioned unrelated to this PR still stands I think those instances should be removed in a separate PR until we have a proper way to tag types so we can request `Monoid<F> @@ Addition` and a package for unsafe instances that require implicit imports.